arrow-udf-js/src/lib.rs

```diff
@@ -243,17 +270,22 @@ impl Runtime {
     ///             }
     ///         }
     /// "#,
+    ///         false,
     ///     )
+    ///     .await
     ///     .unwrap();
+    /// # });
     /// ```
-    pub fn add_function(
+    pub async fn add_function(
         &mut self,
         name: &str,
-        return_type: impl IntoField,
+        return_type: impl IntoField + Send,
```

Comment:
What about making `Send` as a part of `IntoField`?

```rust
pub trait IntoField: private::Sealed + Send {
    fn into_field(self, default_name: &str) -> Field;
}
```

Reply:
Either LGTM. 

It's worth mentioning that the only purpose of this `Send` is to make Rust compiler happy 😄 See this comment:

https://github.com/arrow-udf/arrow-udf/blob/560c39c6b0c81d8298d2dad8e29d62e1c1226fca/arrow-udf-js/src/lib.rs#L108-L115

It's also possible to workaound it by doing `into_field` outside the `async_with!`. The only problem is: [the code looks ugly.](https://github.com/arrow-udf/arrow-udf/blob/346882ed48057b1f09070fa63ac90c62b8ff579c/arrow-udf-js/src/lib.rs#L106-L121)